docs/spec/staking/02_state_transitions.md

```diff
@@ -2,96 +2,114 @@
+- set `validator.Status` to `Bonded`
+- send the `validator.Tokens` from the `NotBondedTokens` to the `BondedPool` `ModuleAccount`
+- delete record the existing record from `ValidatorByPowerIndex`
+- add an new updated record to the `ValidatorByPowerIndex`
+- update the `Validator` object for this validator
+- if it exists, delete any `ValidatorQueue` record for this validator
 
 ### Bonded to Unbonding
-When a validator begins the unbonding process the following operations occur: 
- - update the `Pool` object with tokens moved from `BondedTokens` to `NotBondedTokens`
- - set `validator.Status` to `Unbonding`
- - delete record the existing record from `ValidatorByPowerIndex`
- - add an new updated record to the `ValidatorByPowerIndex`
- - update the `Validator` object for this validator
- - insert a new record into the `ValidatorQueue` for this validator 
+
+When a validator begins the unbonding process the following operations occur:
+
+- send the `validator.Tokens` from the `BondedPool` to the `NotBondedTokens` `ModuleAccount`
+- set `validator.Status` to `Unbonding`
+- delete record the existing record from `ValidatorByPowerIndex`
+- add an new updated record to the `ValidatorByPowerIndex`
+- update the `Validator` object for this validator
+- insert a new record into the `ValidatorQueue` for this validator
 
 ### Unbonding to Unbonded
+
 A validator moves from unbonding to unbonded when the `ValidatorQueue` object
 moves from bonded to unbonded
- - update the `Validator` object for this validator
- - set `validator.Status` to `Unbonded`
 
-### Jail/Unjail 
+- update the `Validator` object for this validator
+- set `validator.Status` to `Unbonded`
+
+### Jail/Unjail
+
 when a validator is jailed it is effectively removed from the Tendermint set.
 this process may be also be reversed. the following operations occur:
- - set `Validator.Jailed` and update object 
- - if jailed delete record from `ValidatorByPowerIndex`
- - if unjailed add record to `ValidatorByPowerIndex`
 
+- set `Validator.Jailed` and update object
+- if jailed delete record from `ValidatorByPowerIndex`
+- if unjailed add record to `ValidatorByPowerIndex`
 
 ## Delegations
 
 ### Delegate
+
 When a delegation occurs both the validator and the delegtion objects are affected  
- - determine the delegators shares based on tokens delegated and the validator's exchange rate
- - remove tokens from the sending account 
- - add shares the delegation object or add them to a created validator object
- - add new delegator shares and update the `Validator` object
- - update the `Pool` object appropriately if tokens have moved into a bonded validator
- - delete record the existing record from `ValidatorByPowerIndex`
- - add an new updated record to the `ValidatorByPowerIndex`
+
+- determine the delegators shares based on tokens delegated and the validator's exchange rate
+- remove tokens from the sending account
+- add shares the delegation object or add them to a created validator object
```

Comment:
???

Reply:
IDK, this was part of the prev staking spec 🤷‍♂